src/Backends/Helmholtz/VLERoutines.cpp

```diff
@@ -613,9 +613,9 @@ void SaturationSolvers::saturation_D_pure(HelmholtzEOSMixtureBackend &HEOS, Cool
         deltaV = rhoV/reduce.rhomolar;
         tau = reduce.T/T;
     }
-    catch(NotImplementedError &e)
+    catch(NotImplementedError &)
     {
-        throw e;
+        throw; // ??? What is this try...catch for?
```

Comment:
I think this was for debugging purposes so I could put a breakpoint here but I forgot to remove it